minigalaxy/api.py

```diff
@@ -145,7 +145,7 @@ def get_download_info(self, game: Game, dlc_installers="") -> tuple:
                 possible_downloads.append(installer)
         if not possible_downloads:
             if operating_system == "linux":
-                return self.get_download_info(game, "windows")
+                return self.get_download_info(game)
```

Comment:
Unfortunately this won't do the trick. The point of this line is to get download_info for windows installer if linux one is not available. This change will put us into infinite loop. We can either:
- Resign from this feature if this is not a real use case (not recommended)
- get_download_info will require a little bit of redesign